tests/unit_tests/client/address_book/impl.cc

```diff
@@ -170,4 +210,19 @@ BOOST_AUTO_TEST_CASE(PGPClearSign) {
 
 // TODO(unassigned): more cases?
 
-BOOST_AUTO_TEST_SUITE_END() 
+BOOST_AUTO_TEST_CASE(RejectDuplicateEntry)
+{
+  // Ensure valid subscription line creates an entry
+  BOOST_CHECK_NO_THROW(AddressBookEntry const entry(subscription.front()));
```

Comment:
>AddressBookEntry const

`const` not necessary here.

Reply:
Right, guess it is a bit too pedantic given the context.